hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AccessLogFilter.java

```diff
@@ -109,13 +113,25 @@ public void filter(ContainerRequestContext requestContext,
             // Record slow query if meet needs, watch out the perf
             if (timeThreshold > 0 && executeTime > timeThreshold &&
                 needRecordLog(requestContext)) {
-                // TODO: set RequestBody null, handle it later & should record "client IP"
-                LOG.info("[Slow Query] execTime={}ms, body={}, method={}, path={}, query={}",
-                         executeTime, null, method, path, uri.getQuery());
+
+                LOG.info("[Slow Query] ip={} execTime={}ms, body={}, method={}, path={}, query={}",
+                         getClientIP(requestContext), executeTime,
+                         requestContext.getProperty(REQUEST_PARAMS_JSON), method, path,
+                         uri.getQuery());
```

Comment:
are they repeated when GET: uri.getQuery() and REQUEST_PARAMS_JSON

Reply:
for GET ， REQUEST_PARAMS_JSON will extract the `id` info from `/example/{id} `;  uri.getQuery() will extract ` id=1` from  `http://example.com/resource?id=1`